mock/mockimpl/kvproc/subdoc.go

```diff
@@ -51,8 +51,10 @@ func (e *Engine) executeSdOps(doc, newMeta *mockdb.Document, ops []*SubDocOp) ([
 		case memd.SubDocOpCounter:
 			opRes, err = SubDocExecutor{doc, newMeta}.executeSdOpCounter(op)
 		case memd.SubDocOpSetDoc:
-		case memd.SubDocOpAddDoc:
+			opRes, err = SubDocExecutor{doc, newMeta}.executeSdOpDictSetFullDoc(op)
 		case memd.SubDocOpDeleteDoc:
+			opRes, err = SubDocExecutor{doc, newMeta}.executeSdOpDeleteFullDoc(op)
+		case memd.SubDocOpAddDoc:
```

Comment:
Is this actually a legitimate command? I've tried using it from a test app against the server and every time I get “incorrectly matched subdocument operation types ”

Reply:
Yea, I think it's not really used and it's instead a SetDoc w/ flags=MkDoc